src/lib/resources.ts

```diff
@@ -55,18 +54,52 @@ export class Vaults extends OPResource {
         return ObjectSerializer.deserialize(data, "Array<Item>");
     }
 
+    /**
+     * Get metadata about a single vault
+     *
+     * @param vaultQuery
+     */
+    public async getVault(vaultQuery: string): Promise<Vault> {
+        if (isValidId(vaultQuery)) {
+            return this.getVaultById(vaultQuery);
+        }
+
+        return this.getVaultByTitle(vaultQuery);
+    }
+
     /**
      * Get metadata about a single vault
```

Comment:
This description now coincides with the one for `getVault`. As a user, I'd ask myself: "What's the difference between them? Their names are different". The parameter may not be enough here.

A description like this should do the trick (feel free to come up with one on your own of you don't like this one 😛 ):
```suggestion
     * Get metadata about a single vault with the provided ID.
```

Reply:
sounds good to me. Changed